jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/taskslist/AbstractTaskListView.java

```diff
@@ -254,6 +286,86 @@ protected void onMouseOverGrid(final ListTable<TaskSummary> extendedPagedTable,
         );
     }
 
+    protected AnchorListItem getBulkClaim(final ExtendedPagedTable<TaskSummary> extendedPagedTable) {
+        final AnchorListItem bulkAbortNavLink = GWT.create(AnchorListItem.class);
+        bulkAbortNavLink.setText(constants.Bulk_Claim());
+        bulkAbortNavLink.setIcon(IconType.LOCK);
+        bulkAbortNavLink.setIconFixedWidth(true);
+        bulkAbortNavLink.addClickHandler(event -> confirmPopup.show(constants.Claim_Confirmation(),
+                                                                    constants.Claim(),
+                                                                    constants.Claim_tasks(),
+                                                                    getClaimCommand(extendedPagedTable))
+        );
+        return bulkAbortNavLink;
+    }
+
+    protected AnchorListItem getBulkRelease(final ExtendedPagedTable<TaskSummary> extendedPagedTable) {
+        final AnchorListItem bulkAbortNavLink = GWT.create(AnchorListItem.class);
+        bulkAbortNavLink.setText(constants.Bulk_Release());
+        bulkAbortNavLink.setIcon(IconType.UNLOCK);
+        bulkAbortNavLink.setIconFixedWidth(true);
+        bulkAbortNavLink.addClickHandler(event -> confirmPopup.show(constants.Release_Confirmation(),
```

Comment:
@nmirasch I think the popup title and the question are inverted in all confirmpopup. Question is showing as the title, can you double check that?

Reply:
@cristianonicolai yes, you are absolutely right! 
updated